Quick_Sort/Quick_Sort.c

```diff
@@ -1,27 +1,34 @@
-#include <bits/stdc++.h>
-
-using namespace std;
+#include <stdio.h>
 
+//Swapping the value in the array 
+void swap(int *a,int *b)
+{
+   int temp= *a;
+   *a = *b;
+   *b = temp;
+}
 int partition (int arr[], int low, int high)
 {
+    //Taking the last value of array as pivot.
     int pivot = arr[high];
     int i = (low - 1);
-    for (int j = low; j <= high- 1; j++)
+    for (int j = low; j <= high- 1 ; j++)
```

Comment:
Fix the spacing - 
- Add space between `high` and `-`
- Remove space between `1` and `;`